examples/xgboost/xgboost_sklearn/conda.yaml

```
@@ -0,0 +1,13 @@
+channels:
+- conda-forge
+dependencies:
+- python=3.8.12
+- pip
+- pip:
+  - mlflow
+  - mypy-extensions==0.4.3
+  - pandas==1.3.4
+  - scikit-learn==0.24.2
+  - typing-extensions==4.0.0
+  - xgboost==1.5.0
```

```suggestion
  - mlflow
  - pandas==1.3.4
  - scikit-learn==0.24.2
  - xgboost==1.5.0
```

Can we remove `mypy-extensions` and `typing-extensions`?